### PR TITLE
Make game decide

### PR DESCRIPTION
**Pull request by mogvaj**

### CODE REVIEW

File: db.py [hunk 1]
```diff
@@ -43,6 +43,13 @@ def addUser(aUserObject):
         cur = con.cursor()
         cur.execute(ins)
 
+def removeUser(aUserName):
+    con = sqlite3.connect(DB_FILE)
+    with con:
+        rem = "DELETE FROM users WHERE userName = '{}'".format(aUserName)
```
Comment (mckib2) [inline db.py:49]: This is susceptible to SQL injection attacks.  Use the [DB-API’s parameter substitution](https://docs.python.org/3/library/sqlite3.html) instead, e.g.
```python
cur.execute("insert into lang values (?, ?)", ("C", 1972))
```

File: db.py [hunk 2]
```diff
@@ -73,13 +80,20 @@ def getRedCards():
...
         cur = con.cursor()
-        cur.execute("select * from users")
+        cur.execute("SELECT * FROM users")
         return cur.fetchall()
 
+def getUserGame(aUserName):
+    con = sqlite3.connect(DB_FILE)
+    with con:
+        cur = con.cursor()
+        cur.execute("SELECT gameCode FROM users WHERE userName = '{}'".format(aUserName))
```
Comment (mckib2) [inline db.py:97]: Ditto

File: db.py (hunk 1 above)
Comment (mogvaj) [reply db.py:49]: How is that any safer?

File: db.py (hunk 2 above)
Comment (mogvaj) [reply db.py:97]: Ditto, too

**Pull request merged**